database/migrations/2021_11_01_213540_change_model_namespaces.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class ChangeModelNamespaces extends Migration
```

Comment:
You also need one for the comments table, which can nbe polymorpthic to: Events, Locations, Participants and Members

Reply:
Thanks, added